controllers/notificationsconfiguration/configmap.go

```diff
@@ -76,10 +86,34 @@ func (r *NotificationsConfigurationReconciler) reconcileNotificationsConfigmap(c
 	// Do nothing
 	return nil
 }
+
 func mapToString(m map[string]string) string {
 	result := ""
 	for key, value := range m {
 		result += fmt.Sprintf("%s: %s\n", key, value)
 	}
 	return result
 }
+
+// checkIfContextEquals checks if context value in NotificationConfiguration and notificationConfigMap context have same value
+// return true if there is difference, and false if no changes observed
+func checkIfContextEquals(cr *v1alpha1.NotificationsConfiguration, notificationConfigMap *corev1.ConfigMap) bool {
```

Comment:
Can we add some unit test for this function?

Reply:
added unit test